.github/actions/go-cache/action.yml

```diff
@@ -13,17 +13,14 @@ runs:
       run: echo 'C:\Program Files\Git\usr\bin' >> $GITHUB_PATH
 
     - name: Load Go cache
+      id: load-go-cache
```

Comment:
single multi purpose cache per platform. Executed for unit tests and E2E CLI tests.